### ISSUE TITLE
@enums false negatives

### ISSUE THREAD

**Issue by zenna** [tonyhffong/Lint.jl#94]
```julia
@enum Z3_lbool Z3_L_FALSE = -1 Z3_L_UNDEF Z3_L_TRUE
```

yields

`[               ] ERROR  Use of undeclared symbol Z3_L_TRUE`

**Comment by tonyhffong** [comment]
Check out the latest master
